.github/workflows/deploy-prod.yml

```diff
@@ -38,10 +38,12 @@ jobs:
         run: |
           HAPPY_OIDC_ID_TOKEN=${{env.id_token}} happy config set API_URL ${{ vars.API_URL }} --env prod --aws-profile ""
       - name: Deploy to production env
-        uses: chanzuckerberg/github-actions/.github/actions/deploy-happy-stack@main
+        uses: chanzuckerberg/github-actions/.github/actions/deploy-happy-stack@v1.26.0
         with:
           tfe-token: ${{ secrets.TFE_TOKEN }}
           env: ${{ vars.HAPPY_ENV }}
-          create-tag: true
```

Comment:
I see we're dropping `create-tag: true` entirely with this PR. I don't really know the specifics of `deploy-happy-stack`, but is this something we'd want to keep with what this PR is adding? It feels like if we wanted a tag created before, we would still want one now, but the new args don't seem to have anything to do with tagging one way or the other. Just wanted to make sure this was purposeful and not an accidentally cut out line!

Reply:
`create-tag` defaults to false

https://github.com/chanzuckerberg/github-actions/blob/0b740d81436af68cb9af1011cab89199823c0161/.github/actions/deploy-happy-stack/action.yml#L11-L14

To use the image promotion, `create-tag` does need to be set to false, so I've just dropped it here and relied on the default.